services/debugger/cache.go

```diff
@@ -0,0 +1,100 @@
+package debugger
+
+import (
+	"github.com/rudderlabs/rudder-server/config"
+	"sync"
+	"time"
+)
+
+/*
+	loadCacheConfig sets the properties of the cache after reading it from the config file.
+This gives a feature of hot readability as well.
+ */
+func (cache *CacheT) loadCacheConfig() {
+	config.RegisterIntConfigVariable(3, &cache.size, true, 1, "LiveEvent.Cache.size")
+	config.RegisterDurationConfigVariable(time.Duration(720), &cache.keyTTL, true, time.Hour,
+		"LiveEvent.cache.ttl")            // default keyTTL is 30 days
+	config.RegisterDurationConfigVariable(time.Duration(15), &cache.cleanupFreq, true, time.Second,
+		"LiveEvent.cache.clearFreq") // default clearFreq is 15 seconds
+}
+
+type cacheItem struct {
+	data       [][]byte
+	lastAccess time.Time
+}
+
+/*
+	CacheT is an in-memory cache. Each key-value pair stored in this cache have a TTL and one goroutine removes the
+key-value pair form the cache which is older than TTL time.
+*/
+type CacheT struct {
+	lock        sync.RWMutex
+	keyTTL      time.Duration // Time after which the data will be expired and removed from the cache
+	cleanupFreq time.Duration // This is the time at which a cleaner goroutines  checks whether the data is expired in cache
+	size        int           // This is the size upto which this cache can store a value corresponding to any key
+	cacheMap    map[string]*cacheItem
+	once        sync.Once
```

Comment:
```suggestion
	KeyTTL      time.Duration // Time after which the data will be expired and removed from the cache
	CleanupFreq time.Duration // This is the time at which a cleaner goroutines  checks whether the data is expired in cache
	Size        int           // This is the size upto which this cache can store a value corresponding to any key
	cacheMap    map[string]*cacheItem
	lock        sync.RWMutex
	once        sync.Once
```

configurable properties should be exportable fields

Reply:
done